ebib-utils.el

```diff
@@ -392,22 +392,38 @@ transformation function returns something that can be displayed."
   :type '(repeat (cons (string :tag "Field")
                        (function :tag "Transform function"))))
 
-(defcustom ebib-TeX-markup-replace-alist '(("{" . "")
-                                           ("}" . "")
-                                           ("``" . "“")
+(defcustom ebib-TeX-markup-replace-alist '(("``" . "“")
 					   ("`" . "‘")
 					   ("''" . "”")
-					   ("'" . "’"))
+					   ("'" . "’")
+					   ("\\\\textit\\(?:\\[.*\\]\\)?{\\([^}]*\\)}"
```

Comment:
The regexes here differ slightly from the original regex in `ebib-clean-TeX-markup`? Did you do that for a reason?

Reply:
Good spot! No, there wasn't a reason. Because of the different method in the code they have to be strucutred slightly differently anyway, so I had to type them out by hand, and I think I just did it a different way. I'll revert to the originals.

Reply:
Done in latest force-push